packages/web-components/src/components/ic-top-navigation/ic-top-navigation.css

```diff
@@ -245,6 +245,10 @@ slot[name="toggle-icon"] svg {
     min-height: 2.5rem;
   }
 
+  .app-details-container {
+    margin-left: var(--ic-space-xs);
```

Comment:
The app details container was designed to be positioned 8px from the left in mobiles. This change will mean that the extra small top navigation will not look as the designers intend

Reply:
Please ignore the comment above. I've spoken to the design team who are happy with pushing the app title to the right 4px